### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by shawndwells**
Creates a table to show what each sample event does

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,24 @@
 
 [![Container Repository on Quay](https://quay.io/repository/crowdstrike/detection-container/status "Container Repository on Quay")](https://quay.io/repository/crowdstrike/detection-container)
 
-This container will create detections and preventions on a Linux container protected by a CrowdStrike sensor.
+This container will create detections and preventions on a Linux container platform protected by a CrowdStrike sensor.
```
Comment (redhatrises) [inline README.md:5]: What do you mean by a `Linux container platform`? I can run this in the container on a linux host, right?

**Comment by shawndwells** [comment]
Yes. That means installing some container engine and turning the host into
a container platform though

**Comment by shawndwells** [comment]
Hmm, what really trying to get across is that a Linux container on a
windows host won't generate detections yet  . Is there a better way to
express that?

**Comment by redhatrises** [comment]
What about this? 

```
This container will create detections and preventions only on Linux hosts protected by a CrowdStrike sensor.
```

**Comment by shawndwells** [comment]
> What about this?
> 
> ```
> This container will create detections and preventions only on Linux hosts protected by a CrowdStrike sensor.
> ```
> 
> Edit:
> 
> Can also add: `This is not supported on Windows` somewhere

But this isn't about Linux hosts (aka virtual machines), it's about Linux container platforms like docker or openshift or coreos

File: README.md (hunk 1 above)
Comment (shawndwells) [inline README.md:5]:
```suggestion
This container will create detections and preventions only on Linux hosts and container platforms protected by a CrowdStrike sensor.
```

**Comment by redhatrises** [comment]
Thanks! Ack

**Pull request merged**